stylesheets/_component.rules.scss

```diff
@@ -298,3 +298,306 @@ td.table__form .rules + .form__group:last-of-type {
+    .rule-block--and + .rule-links .rule-link--or {
+        border-left: 4px solid color(gray, lightest);
+        margin-left: 20px;
+        padding-left: 40px;
+
+        &::after {
+            left: 44px;
+        }
+    }
+
+    .rule-link--or + .rule-link--and {
+        padding-top: 0;
+
+        &::after {
+            top: 7px;
+        }
+    }
+
+    .rule-link--and {
+        border-left: 4px solid color(gray, lightest);
+        margin-left: 20px;
+
+        button {
+            background-color: lighten(#e8b750, 30);
+        }
+    }
+
+    .rule-link--else-if {
+        margin-left: -20px;
+        padding-top: 20px;
+        padding-bottom: 20px;
+
+        &::before {
+            border-top: 2px dashed color(gray, lightest);
+            content: '';
+            display: block;
+            width: 100%;
+            height: 3px;
+            position: absolute;
+            top: 38px;
+        }
+
+        &::after {
+            top: 27px;
+        }
+
+        button {
+            background-color: lighten(#3f8ccc, 40);
+            margin-left: 30px;
+            min-width: 76px;
+        }
+    }
+
+    .rule-block + .rule-block,
+    .rule-block--indent > .rule-block {
+        margin-top: -1px;
+
+        // visually distinguish blocks close together
+        .rule-block__header {
+            border-top: 1px solid rgba(255, 255, 255, .5);
+        }
+    }
+
+    // first block, indented one spot
+    .rule-block + .rule-block--return,
+    .rule-block--if + .rule-block--or {
+        margin-left: 40px;
+        width: calc(100% - 40px);
+    }
+
+    .rule-block--indent > .rule-links,
+    .rule-block--indent > .rule-block--indent + .rule-links {
+        padding-left: 20px;
+    }
+
+    .rule-block--indent {
+        border-left: 4px solid color(gray, lightest);
+        margin-left: 20px;
+        padding-left: 20px;
+    }
+
+    .rule-block__header {
+        border: 0;
+        box-shadow: none;
+        color: color(white);
+        font-family: $font-family-regular;
+        font-size: 18.66px; // For AA large text
+        padding: .5em;
+
+        @include respond-min($screen-tablet) {
+            padding: 0;
+        }
+    }
+
+    .rule-block__heading {
+        padding: 0;
+    }
+
+    .rule-block--if,
+    .rule-block--else-if {
+        background-color: lighten(#3f8ccc, 40);
+        border-color: #3f8ccc;
+        border-top-left-radius: 15px;
+
+        @include respond-min($screen-tablet) {
+            border-top-right-radius: 15px;
+        }
+
+        .rule-block__header {
+            background-color: #3f8ccc;
+            border-top-left-radius: 14px;
+        }
+
+        .remove-control {
+            color: color(white);
+
+            @include respond-min($screen-tablet) {
+                color: color(text);
+            }
+        }
+    }
+
+    .rule-block--else-if {
+        &::after {
+            border-top: 4px solid color(gray, lightest);
```

Comment:
alpha order

Reply:
Can we get a tool to automate this? Seems trivial?

Reply:
probably something we can add to the linter somehow

Reply:
Pretty sure sass linter has this but we've never turned it on due to older code.